Source/Fuse.Storage/UserSettingsModule.uno

```diff
@@ -261,7 +299,12 @@ namespace Fuse.Storage
 		*/
 		object Clear(Context c, object[] args)
 		{
-			_userSetting.Clear();
+			if defined(Android)
+					AndroidUserSettingsImpl.Clear();
+				else if defined(iOS)
+					IOSUserSettingsImpl.Clear();
+				else
+					DesktopUserSettingsImpl.GetInstance().Clear();
```

Comment:
I think most of this hunk accidentally got indented a bit too deep :)

Reply:
ops! fix underway.